pkgs/development/tools/misc/stlink-tool/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, pkg-config
+, fetchFromGitHub
+, libusb1
+}:
+
+# IMPORTANT: You need permissions to access the stlink usb devices.
+# Add services.udev.packages = [ pkgs.stlink ] to your configuration.nix
+stdenv.mkDerivation rec {
+  pname = "stlink-tool";
+  version = "unstable-2023-10-20";
+
+  src = fetchFromGitHub {
+    owner = "UweBonnes";
+    repo = pname;
+    rev = "3ab0b140561c18c6e51183877c849b7b8ca0319c";
+    sha256 = "sha256-Wj72kHckxnox0kfqEZzE5zeSi8Bc+QSHPs4BtIf6RhE=";
```

Comment:
```suggestion
    hash = "sha256-Wj72kHckxnox0kfqEZzE5zeSi8Bc+QSHPs4BtIf6RhE=";
```

Reply:
Rebased of the original (non-fork) repo, so obsolete now.